assets/themes/base16.tmTheme

```diff
@@ -3,14 +3,21 @@
 <plist version="1.0">
     <dict>
         <!--
+        This theme is meant for users who configure their terminal with a base16
+        color theme. If you use a 256-color variant along with base16-shell, use
+        the bat theme base16-256 instead.
+
+        base16: https://github.com/chriskempson/base16
+        base16-shell: https://github.com/chriskempson/base16-shell
+
         The colors in this theme are encoded as #RRGGBBAA where RR is an ANSI
-        palette number from 00 to 0f, and AA is the special value 00 to indicate
-        that this encoding is being used.
+        palette number from 00 to 0f (the 8 colors and their bright versions),
+        and AA is the special value 00 to indicate this encoding is being used.
         -->
         <key>author</key>
         <string>Template: Chris Kempson, Scheme: Mitchell Kember</string>
         <key>name</key>
-        <string>Base16</string>
+        <string>Base16 256</string>
```

Comment:
Shouldn't this one remain `Base16` and the other one be named `Base16 256`?

Reply:
Yes, thanks for catching that. Fixed.